gui/tools/Helper.h

```diff
@@ -0,0 +1,227 @@
+      }
+      return ret;
+    }
+    else
+    {
+      return "";
+    }
+  }
+
+  //----------------------------------------------------------------------------
+  // Description
+  // Read global path and extract all contained path
+  static std::vector<std::string> ExtractAllFilePath(const char* globalPath)
+  {
+    std::vector<std::string> pathList;
+
+    // Open file which contains the list of all file
+    std::ifstream container(globalPath);
+    if (!container.is_open())
+    {
+      std::cerr << "Unable to open : " << globalPath << std::endl;
+      return pathList;
+    }
+
+    // Extract path of globalPath from globalPath
+    //std::string pwd_str = globalPath;
+    std::string directoryPath = help::GetFilenamePath(std::string(globalPath));
+    // Get current working directory
+    if (directoryPath == "")
+    {
+      directoryPath = vtksys::SystemTools::GetCurrentWorkingDirectory();
+    }
+
+    std::string path;
+    while (!container.eof())
+    {
+      std::getline(container, path);
+      // only get the file name, not the whole path
+      std::vector <std::string> elems;
+      help::SplitString(path, ' ', elems);
+
+      // check if there are an empty line
+      if (elems.size() == 0)
+      {
+        continue;
+      }
+
+      // Create the real data path to access depth map file
+      pathList.push_back(directoryPath + "/" + elems[elems.size() - 1]);
+    }
+
+    return pathList;
+  }
+
+  //----------------------------------------------------------------------------
+  // Description
+  // Read global path and extract all contained path
+  static std::vector<std::string> ExtractAllKRTDFilePath(const char* globalPath, const char* framelist)
+  {
+    std::vector<std::string> pathList;
+
+    // Open file which contains the list of all file
+    std::ifstream container(framelist);
+    if (!container.is_open())
+    {
+      std::cerr << "Unable to open : " << framelist << std::endl;
+      return pathList;
+    }
+
+    std::string path;
+    while (!container.eof())
+    {
+      std::getline(container, path);
+      // only get the file name, not the whole path
+      std::vector <std::string> elems;
+      help::SplitString(path, ' ', elems);
+      // check if there are an empty line
+      if( elems.empty() )
+      {
+        continue;
+      }
+      std::vector <std::string> filename;
+      help::SplitString(elems[elems.size() - 1], '/', filename);
+
+      // Create the real data path to access depth map file
+      std::vector <std::string> elemsWithoutExtension;
+      help::SplitString(filename[filename.size() - 1], '.', elemsWithoutExtension);
+      pathList.push_back(std::string(globalPath) + "/" + elemsWithoutExtension[0] + ".krtd");
+    }
+
+    return pathList;
+  }
+
+  //----------------------------------------------------------------------------
+  // Description
+  // Read krtd file and create K and RT matrix
+  static bool ReadKrtdFile(std::string filename, vtkMatrix3x3* matrixK,
```

Comment:
I've got this warning too, but it's used in ReconstructionData (ReconstructionData.cxx:74), which is used by MeshColoration, which is used by ColorizeSurfaceOptions. When I comment this function in Helper.h, I have build errors. 
Anyway, this function (and I assume the other ones you pointed at, even though I haven't checked yet) should disappear because they're duplicate of already existing ones.